include/sound/hdaudio_ext.h

```diff
@@ -82,7 +82,7 @@ struct hdac_ext_stream {
 
 int snd_hdac_ext_stream_init_all(struct hdac_bus *bus, int start_idx,
 				 int num_stream, int dir);
-void snd_hdac_stream_free_all(struct hdac_bus *bus);
+void snd_hdac_ext_stream_free_all(struct hdac_bus *bus);
```

Comment:
nitpick for the commit title: it is not really adding hdac_ext as prefix, but replaces the 'hdac' with 'hdac_ext'.
"Use snd_hdac_ext prefix in snd_hdac_stream_free_all() for clarity" perhaps?

Reply:
done.